source/quickstart/container.rst

```diff
@@ -6,35 +6,303 @@ Quickstart for Containers
 
 .. default-domain:: minio
```

Comment:
This is a heck of a job, but we're also playing with a *lot* of tabs and sections here.

We had I think originally talked about possibly breaking up the Containers section, and I'm starting to come around to that.

Maybe it makes sense to have `quickstart/docker.rst` and `quickstart/podman.rst` - we're duplicating some instructions, but the *flow* will be easier for users. 

Let me know what you think.

Reply:
AB was interested in mimicking the Download page on the website.
It has a `Docker` tab that just uses `podman` (no Docker instructions).

I suggest we leave it is a single page.
Maybe reconsider including Docker as an option?

Reply:
I thought i commented but github says no.

Docker still has the lions share of the market and mind, so we have to address it alongside Podman for the forseeable future.